benchmarks/src/main/kotlin/benchmarks/common/BenchmarkUtils.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2016-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package benchmarks.common
+
+import java.util.concurrent.*
+
+fun doGeomDistrWork(work: Int) {
+    // We use geometric distribution here
```

Comment:
Let's add a comment like `We also checked on XXX machine that the resulting work times are distributed geometrically, see <link_to_your_github_comment_with_the_plot>.`